resources/js/components/FormField.vue

```diff
@@ -75,6 +75,7 @@ export default {
   components: { Draggable },
 
   props: ['resourceName', 'resourceId', 'field'],
+  
```

Comment:
PR-is pigem ei ole welcome stiilimuudatused, unnecessary tühi rida.

Reply:
Git merge-iga tekkisid vead.